### PR TITLE
Add more charts to place page

### PR DESCRIPTION
**Pull request by shifucun**
First batch adding the following topics:

1. Literacy
   * Count_Person_Literate
   * Count_Person_Illiterate

2. Social Category
   * Count_Person_ScheduledCaste
   * Count_PersonScheduledTribe

3. Households
   * Count_Household_Rural
   * Count_Household_Urban

4. Poverty
   * Count_Person_Rural_BelowPovertyLevelInThePast12Months
   * Count_Person_Urban_BelowPovertyLevelInThePast12Months

5. Homelessness
   * Count_Person_Houseless_Urban
   * Count_Person_Houseless_Rural

6. Employment
   * Count_Person_NonWorker
   * Count_Person_Worker

### CODE REVIEW

**Comment by pradh** [comment]
Thanks for adding these! A screenshot of how it looks might be nice...

File: server/chart_config.json [hunk 1]
```diff
@@ -57,6 +57,22 @@
     "isOverview": true,
     "isChoropleth": true
   },
+  {
+    "category": "Economics",
+    "titleId": "Worker vs non-worker population",
+    "title": "Worker vs non-worker population",
```
Comment (pradh) [inline server/chart_config.json:63]: non-worker is one word https://www.thefreedictionary.com/nonworker

"Worker vs. Nonworker Population"

?

File: server/chart_config.json [hunk 2]
```diff
@@ -57,6 +57,22 @@
...
+    "description": "Worker vs non-worker population",
+    "statsVars": ["Count_Person_Workers", "Count_Person_NonWorker"],
+    "relatedChart": {
+      "titleId": "Percentage of worker and non-worker",
+      "title": "Percentage of worker and non-worker",
```
Comment (pradh) [inline server/chart_config.json:68]: Percentage of Workers and Nonworkers

File: server/chart_config.json (hunk 1 above)
Comment (pradh) [reply server/chart_config.json:63]: Title-case titles other places too?

File: server/chart_config.json [hunk 3]
```diff
@@ -1142,6 +1210,38 @@
       "scale": false
     }
   },
+  {
+    "category": "Demographics",
+    "titleId": "Literacy",
+    "title": "Literacy",
+    "description": "Number of literate and illiterate population",
```
Comment (pradh) [inline server/chart_config.json:1217]: Just "Literate and Illiterate Population"  (i.e., No "Number of")

File: server/chart_config.json [hunk 4]
```diff
@@ -1142,6 +1210,38 @@
...
+      "title": "Percentage of literate and illiterate population",
+      "scaling": 100,
+      "unit": "%"
+    }
+  },
+  {
+    "category": "Demographics",
+    "titleId": "Social Category",
+    "title": "Social Category",
+    "description": "Number of population by social category",
```
Comment (pradh) [inline server/chart_config.json:1233]: here too... drop "Number of" ?

**Review by pradh — APPROVED** [review]

File: server/chart_config.json (hunk 1 above)
Comment (shifucun) [reply server/chart_config.json]: The place page has not been using title case. I will not change it for now. If needed, we can do an overall conversion.

File: server/chart_config.json (hunk 2 above)
Comment (shifucun) [reply server/chart_config.json]: Done

File: server/chart_config.json (hunk 3 above)
Comment (shifucun) [reply server/chart_config.json]: Done

File: server/chart_config.json (hunk 4 above)
Comment (shifucun) [reply server/chart_config.json]: Done

**Comment by shifucun** [comment]
Thanks for review!

**Pull request merged**